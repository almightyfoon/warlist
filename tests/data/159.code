c5501b_-0hb-2w2w2w2w2w2w2w2wd_3y3y3y4LhMhM___-0zdUhI0v3y3y3LdPfW35363u35363u3x3xhT