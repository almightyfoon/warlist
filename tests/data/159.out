https://conflictchamber.com/#c5501b_-0hb-2w2w2w2w2w2w2w2wms3y3y3y4LhMhM___-0zdUhI0v3y3y3LdPfW35363u35363u3x3xhT

Retribution Army - 68 / 75 points
[Theme] Forges of War

[Vyros 2] Vyros, Incissar of the Dawnguard [+27]
 - Griffon [8]
 - Griffon [8]
 - Griffon [8]
 - Griffon [8]
 - Griffon [8]
 - Griffon [8]
 - Griffon [8]
 - Griffon [8]
 - Imperatus [20]
Arcanist Mechanik [2]
Arcanist Mechanik [2]
Arcanist Mechanik [2]
House Shyeel Artificer [5]
House Shyeel Arcanists [0(7)]
House Shyeel Arcanists [0(7)]

Retribution Army - 75 / 75 points
[Theme] Defenders of Ios

[Issyria 1] Issyria, Sibyl of Dawn [+29]
 - Hemera [16]
 - Manticore [14]
Arcanist Mechanik [2]
Arcanist Mechanik [2]
Fane Knight Skeryth Issyen [0(8)]
Houseguard Thane [0(4)]
Lys Healer [3]
Houseguard Halberdiers (max) [13]
 - Houseguard Halberdier Officer & Standard [0(4)]
 - Soulless Escort (1) [1]
Houseguard Halberdiers (max) [13]
 - Houseguard Halberdier Officer & Standard [4]
 - Soulless Escort (1) [1]
Stormfall Archers [9]
Stormfall Archers [9]
Arcantrik Force Generator [17]
